NewsScraper/DawnCrawler/DawnSpiders/spiders/dawn.py

```diff
@@ -0,0 +1,61 @@
+from DawnSpiders.items import ArticleItem
+from scrapy.linkextractors import LinkExtractor
+from scrapy.spiders import CrawlSpider, Rule
+
+
+class DawnSpider(CrawlSpider):
+    name = 'dawn'
+    allowed_domains = ['dawn.com']
+    start_urls = [
+        "https://www.dawn.com/"
+
+    ]
+    rules = (
+        Rule(
+            LinkExtractor(restrict_xpaths=[
```

Comment:
Why not use restrict css? All these can be  combined into `[class^='nav__item  nav__item--']` one selector.

Reply:
I realized I could use nav__item nav__item-- and that it would save time but the reason I decided not to use it is because it seemed to general to me, I thought they could use these same tags for some other list in the future, I was thinking specific is better then general